ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 3.1.38 (in development)
 -----------------------
+- The restriction preventing the use of dynamic linking in combination with
```

Comment:
```suggestion
- Remove the restriction preventing the use of dynamic linking in combination with
```

Reply:
Added missing "was removed".  IIUC changelog entries are better phrase in that way.  e.g. `The restriction  .. was removed`